wallet/README.md

```diff
@@ -28,8 +28,8 @@ itself to restore from the backup, see the separate [README.recover.md](README.r
 The current fee rate for each of the fee categories (economic, normal, priority) is cached in
 app-private storage:
 
-    Mainnet: /data/data/de.schildbach.wallet/files/fees.txt
-    Testnet: /data/data/de.schildbach.wallet_test/files/fees-testnet.txt
+    Mainnet: /data/data/hashengineering.groestlcoin.wallet/files/fees.txt
+    Testnet: /data/data/hashengineering.groestlcoin.wallet_test/files/fees-testnet.txt
```

Comment:
can you check if this needs to be `hashengineering.groestlcoin.wallet` or `de.schildbach.wallet` ? I am not able to find either locations on my android under Android/data/data/

Reply:
It is this `hashengineering.groestlcoin.wallet` (based on the package of the app) and it won't be visible because it is private app data.

Reply:
Can you change that in the readme then?